js/background/background.js

```diff
@@ -1,9 +1,8 @@
 class CacheStorage {
-    static timestamp() { return Math.trunc(Date.now() / 1000); }
     static isExpired (timestamp, ttl) {
         if (!timestamp) return true;
         if (typeof ttl != 'number' || ttl < 0) ttl = 0;
-        return timestamp + ttl <= CacheStorage.timestamp();
+        return timestamp + ttl <= window.timestamp();
```

Comment:
This works? Where can I find info on this API.

Reply:
That's actually a call to the function declared in core.js.
The window object holds all global functions